### PR TITLE
Add influxdb auth

### PR DESCRIPTION
**Pull request by UnderGreen**

### CODE REVIEW

**Comment by UnderGreen** [comment]
creating user is a safe operation, if user exists returned code will be still zero

**Comment by UnderGreen** [comment]
@eldios can you take a look on this PR?

File: resources/keeper.yaml [hunk 1]
```diff
@@ -186,3 +196,13 @@ spec:
...
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: telegraf-influxdb-creds
+  namespace: default
+data:
+  username: dGVsZWdyYWY=
+  password: dGVsZWdyYWY=
```
Comment (eldios) [inline resources/keeper.yaml]:
```
password: c3VwZXJTZWN1cmVQYXNzd29yZAo=
```
I think

Comment (UnderGreen) [inline resources/keeper.yaml]: For sure, but should be `c3VwZXJTZWN1cmVQYXNzd29yZAo=`.
Your password after decoding will be with trailing newline and will not work normally. Always remember about trailing newlines. For, example, if u create base 64 encoded password with echo, use `echo -n "password" | base64`. :smiley:

Comment (eldios) [inline resources/keeper.yaml]: true! good catch :+1:

**Pull request merged**